src/main/resources/user/walkthrough/Filtering.md

```diff
@@ -107,15 +107,17 @@ EQUAL will only return Entities and Edges with identifiers that match the seed e
 - if you provide an Edge seed, you will only get back Edges that have the same source, destination and directed values.
 
 RELATED will return the EQUAL results (as above) and additional Entities and Edges:
-- if you provide an Entity seed, you will also get back Edges that have have the same vertex value as source or destination.
-- if you provide an Edge seed, you will also get back Entities that have have the same source or destination vertices.
+- if you provide an Entity seed, you will also get back Edges that have have the same source or destination as the vertex value.
+- if you provide an Edge seed, you will also get back Entities that have have the same vertex value as the source or destination.
```

Comment:
Repeated word 'have' on both lines. This typo already existed but this is a chance to fix it.
The paragraph above also has similar typos.